geetools/ee_image_collection.py

```diff
@@ -896,6 +905,12 @@ def groupInterval(self, unit: str = "month", duration: int = 1) -> ee.List:
             lambda dr: ic.filterDate(ee.DateRange(dr).start(), ee.DateRange(dr).end())
         )
 
+        def add_size(ic):
+            ic = ee.ImageCollection(ic)
+            return ic.set({sizeName: ic.size()})
+
+        imageCollectionList = imageCollectionList.map(add_size).filter(ee.Filter.gt(sizeName, 0))
```

Comment:
by doing this the extra property remains in each images right ? 
I'm not sure we can do without the double mapping and I don't want geetools to leave a trace in the final output, if we are lucky GEE is clever enough to make it quick.